bot_code/modelHelpers/actions/dynamic_action_handler.py

```diff
@@ -27,11 +31,13 @@ class DynamicActionHandler(SplitActionHandler):
     combo_name_index_map = {}
     action_sizes = []
     combo_action_sizes = []
+    actions = []  # Array of actions.

```

Comment:
This is in the super class idk if we need to redefine it

Reply:
Moved some thing to the super class